Prose/ProseLib/Sources/ConversationFeature/MessageBar/TypingIndicator.swift

```diff
@@ -0,0 +1,205 @@
+//
+// This file is part of prose-app-macos.
+// Copyright (c) 2022 Prose Foundation
+//
+
+import AppLocalization
+import Assets
+import ProseCoreTCA
+import SwiftUI
+
+private let l10n = L10n.Content.MessageBar.self
+
+// TODO: Find a better name
```

Comment:
@nesium How could I name this?

Reply:
`Name` maybe?

Reply:
I assume that we'll just have a `User` there at some point. Say a `Chat` had a dictionary `let participants: [JID: User]` and the `participantStates` contains the `JID`, we'd look up the matching participant and would take the name from there. So, right now I don't think we need to worry about the somewhat weird name.